### PR TITLE
Doc: update documentations for SDFT

### PR DESCRIPTION
**Pull request by Qianruipku**
1. update documentations for SDFT for issue #1291 
2. change default value of initsto_freq

### CODE REVIEW

**Review by wenfei-li — APPROVED** [review]

File: source/Makefile [hunk 1]
```diff
@@ -1,153 +1,177 @@
 # This is the Makefile of ABACUS
-
-ifdef GNU
-    include Makefile.gnu
+include Makefile.vars
+#==========================
+# Compiler information 
+#==========================
+INCLUDES = -I. -Icommands -I../
+LIBS = -lifcore -lm -lpthread
```
Comment (caic99) [inline source/Makefile:7]: `-lifcore` is the runtime library for Intel fortran. Is it necessary?

File: source/Makefile [hunk 2]
```diff
@@ -1,153 +1,177 @@
...
-:./module_relaxation\
-:./src_io\
-:./src_parallel\
-:./src_pdiag\
-:./src_pdiag/MRRR\
-:./src_ri\
-:./\
+ifeq ($(DEBUG), ON)
+    INTEL = OFF
+    HONG += -D__DEBUG
```
Comment (caic99) [inline source/Makefile:31]: Where was this flag defined in code? Defining `NDEBUG` is standard.

**Review by caic99 — COMMENTED** [review]
This PR contains codes modifications, and should not be tagged as "docs".

File: source/Makefile (hunk 1 above)
Comment (Qianruipku) [reply source/Makefile:7]: Old Makefile has it. I only keep it. If it is useless, it can be deleted.

File: source/Makefile (hunk 2 above)
Comment (Qianruipku) [reply source/Makefile:31]: It is not used now. However, we should define a HONG for debugging and maybe it is not the name "__DEBUG"

**Pull request merged**